dataplane/src/drivers/kernel.rs

```diff
@@ -274,14 +273,14 @@ impl DriverKernel {
         };
 
         // Spawn workers
-        let (to_workers, from_workers) = match Self::spawn_workers(workers, setup_pipeline) {
+        let (to_workers, from_workers) = match Self::spawn_workers(num_workers, setup_pipeline) {
             Ok(chans) => chans,
             Err(e) => {
                 error!("Failed to start workers: {e}");
                 return;
             }
         };
-        let buckets = to_workers.len().max(1);
```

Comment:
I disagree with this change as if `num_workers` is different than `to_workers.len()` we'll panic on the index of `to_workers` below.  I'll push a fix since I don't quite understand why the `max(1)` is needed either.

Reply:
Fixed.